ext/RMagick/rmimage.c

```diff
@@ -7747,11 +7752,21 @@ Image_get_pixels(VALUE self, VALUE x_arg, VALUE y_arg, VALUE cols_arg, VALUE row
         color.green = GetPixelGreen(image, pixels);
         color.blue  = GetPixelBlue(image, pixels);
         color.alpha = GetPixelAlpha(image, pixels);
+        color.black = GetPixelBlack(image, pixels);
         rb_ary_store(pixel_ary, n, Pixel_from_PixelPacket(&color));
 
         pixels += GetPixelChannels(image);
 #else
-        rb_ary_store(pixel_ary, n, Pixel_from_PixelPacket(&pixels[n]));
+        MagickPixel mpp;
```

Comment:
Do we also need this logic inside `Image_pixel_color`? (https://github.com/rmagick/rmagick/blob/15099295a2b3928eec7bfc1fa6daeab3fcbb06e7/ext/RMagick/rmimage.c#L10728)